package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "novoda.github.io",
+  "version": "1.0.0",
+  "description": "Novoda Open Source Landing Page https://novoda.github.io",
+  "main": "gulpfile.js",
+  "dependencies": {},
+  "devDependencies": {
+    "gulp": "^3.9.1",
+    "gulp-coffeelint": "^0.6.0",
+    "gulp-concat": "^2.6.1",
+    "gulp-uglify": "^2.0.0",
+    "gulp-coffee": "^2.3.3",
+    "gulp-util": "^3.0.8",
+    "gulp-if": "^2.0.2",
+    "gulp-watch": "^4.3.11"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/novoda/novoda.github.io.git"
```

Comment:
@tasomaniac I am not an expert on this thing neither, I have never seen this notation before. "https://blabla.git". I am used to simply see `https://` or `git://` within the `repository:url` of a `package.json` for `npm`. 

Is this notation correct? I am curious.